src/style.css

```diff
@@ -1,3 +1,91 @@
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p {
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+ul,
+ol {
+  padding-left: 0;
+  margin-top: 0;
+  margin-bottom: 0;
```

Comment:
Similar to the previous comment, be consistent with your vertical margins for lists. Choose either top or bottom margin to apply to your elements.